Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -224,10 +237,34 @@ def examineFiles(self, files):
             for tag in subseriesTags:
                 value = slicer.dicomDatabase.fileValue(file, self.tags[tag])
                 value = value.replace(",", "_")  # remove commas so it can be used as an index
+
                 if tag not in subseriesValues:
                     subseriesValues[tag] = []
-                if not subseriesValues[tag].__contains__(value):
+
+                if tag in vectorTags:
+                    if value != "":
+                        element = numpy.array([float(i) for i in value.split("\\")])
+
+                        found = False
+                        for subseriesValue in subseriesValues[tag]:
+                            subseriesElement = numpy.array([float(i) for i in subseriesValue.split("\\")])
```

Comment:
Sorry, I wasn't clear : o

What I meant was that each part of the vector that was previous a cosine could be more generally be called an element, but the list of elements would be a vector.  So my suggestion would be

`vector = numpy.array([float(i) for i in value.split("\\")])`
 
and since we aren't ever working with the individual components of the vector I guess we don't need the word element after all.

Reply:
Sure, not a problem. changed in https://github.com/Slicer/Slicer/pull/7362/commits/64b037eb264de0d6f87c4b0aa79e49bd399176ca 

as 

`vector = numpy.array([float(element) for element in value.split("\\")])`

Reply:
I added element word in the split (: let me know if it is ok (: